tee-worker/litentry/core/stf-task/receiver/src/stf_task_receiver.rs

```diff
@@ -108,6 +119,21 @@ where
 						}
 					}
 				},
+				Assertion::A4 => {
+					let from_date = "2022-10-16T00:00:00Z".to_string();
+					#[cfg(feature = "clock")]
```

Comment:
where is the `clock` feature used?

Reply:
it's for `Utc::now()` within chrono-sgx compile condition.

Reply:
I mean we never activated this feature in Cargo.toml for `chrono-sgx`? `clock` is not included as default features if I'm not mistaken